lib/screens/v2/widget_instance/departures.ex

```diff
@@ -414,9 +415,12 @@ defmodule Screens.V2.WidgetInstance.Departures do
        ),
        do: %{time: %{type: :icon, icon: :overnight}}
 
+  # credo:disable-for-lines:30 Credo.Check.Refactor.CyclomaticComplexity
```

Comment:
I'm not sure I've ever seen this check really be useful, in terms of the subjective experience of reading/understanding/maintaining the function... I wouldn't complain if we disabled it entirely, and it looks like there is precedent for this on other projects. If we don't do that, though, I wonder if this would work with just `disable-for-next-line`, as the single line with the `defp` is the one the issue is reported on (and then we don't have to deal with the line count inevitably falling out of sync).

Reply:
@digitalcora ah! i thought `disable-for-next-line` wasn't working there earlier so i changed it to the `for-lines:30`... turns out i had just typoed `disable-for-next-line`. fixed!

Reply:
if we'd like to disable this altogether (i'm in favor of that!): https://github.com/mbta/screens/pull/2112